src/styles/components/messages.less

```diff
@@ -14,17 +14,15 @@
 **/
 @import "./../palette.less";
 
-.notification-message {
-  z-index: 1111;
-  min-width: 35em;
-}
 
 .notification-message-container {
   position: absolute;
   top: 1em;
   right: 1em;
-  &.login {
+  &.login,
+  .notification-message{
     z-index: 1111;
```

Comment:
Is there some rationale behind this very large z-index? I don't know much about z-indexes, but this looks like some special value.

Reply:
its what was there before... not a great answer I know

Reply:
I tried again with a z-index of 20, and it works for the login page... going to check the other places its used